test/e2e/tests/edit-gas-fee.spec.js

```diff
@@ -0,0 +1,280 @@
+const { strict: assert } = require('assert');
+const {
+  connectDappWithExtensionPopup,
+  getWindowHandles,
+  largeDelayMs,
+  withFixtures,
+  regularDelayMs,
+} = require('../helpers');
+
+describe('Editing Confirm Transaction', function () {
+  it('allow selecting high, medium, low gas estimates on edit gas fee popover', async function () {
+    const ganacheOptions = {
+      hardfork: 'london',
+      accounts: [
+        {
+          secretKey:
+            '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+          balance: '0x15af1d78b58c40000',
```

Comment:
Might be nice to see if we can still represent this as a decimal, otherwise we will have to convert this by some other means.

Reply:
done 👍